crew

```diff
@@ -211,8 +211,19 @@ def generate_compatible
 end
 
 def search (pkgName, silent = false)
-  Find.find (CREW_LIB_PATH + 'packages') do |filename|
-    return set_package(pkgName, filename) if filename == CREW_LIB_PATH + 'packages/' + pkgName + '.rb'
+  return set_package(pkgName, "#{CREW_LIB_PATH}/packages/#{pkgName}.rb") if File.exist?("#{CREW_LIB_PATH}/packages/#{pkgName}.rb")
+  Find.find ("#{CREW_LIB_PATH}/packages/") do |filename|
+    if File.extname(filename) == '.rb'
+      packageName = File.basename filename, '.rb'
+      case packageName
+      when 'hunspell', 'php', 'imagemagick'
```

Comment:
This "list" should be referenced outside of `crew`.  There are other places in `crew` we could abstract like this also.